components/NavBar.vue

```diff
@@ -1,7 +1,25 @@
+<script setup lang="ts">
+const user = useSupabaseUser();
+const supabase = useSupabaseClient();
+
+const handleLogOut = async (): Promise<void> => {
+  const { error } = await supabase.auth.signOut();
+  navigateTo('/');
```

Comment:
```suggestion
  return navigateTo('/');
```

https://nuxt.com/docs/api/utils/navigate-to